### DISCUSSION TITLE
UI freezes when converting DOM to PNG image

### DISCUSSION THREAD

**Discussion by karan182020** [1904labs/dom-to-image-more#150] [Q&A]
I am using dom-to-image-more 3.1.6 in my angular project. It is able to download the image for dom element (Gantt chart) without timeline elements in my project. But when there are timeline elements in the Gantt chart, **UI freezes for more than 20 seconds** when I tried to download it as the DOM size increases.

Is it a known issue or what is the suggestion to fix this issue? 

**Note:** I already tried to run this with **async and await** but it did not work.

**Comment by IDisposable** [comment]
Would need a JSFiddle to see what's happening, but sounds like an enormous DOM... you might want to try the post-filtering work done by @zm-cttae here https://github.com/zm-cttae/dom-inline-style-filter
